demisto_sdk/commands/download/downloader.py

```diff
@@ -341,6 +341,15 @@ def filter_custom_content(
                 or (compiled_regex and re.match(compiled_regex, content_item_name))
                 or content_item_name in self.input_files
             ):
+                # XSUP-39266: replacing playbookId with playbookName in tasks.
```

Comment:
let's move everything to a method: `format_playbook_tasks`

Reply:
also this method should be called **after** filter_custom_content (we shouldn't format inside a method that does filtering)